helm/templates/_helpers.tpl

```diff
@@ -198,4 +198,46 @@ storage:
 {{- end }}
   limits:
     maxDownloadMBs: 10
-{{- end }}
\ No newline at end of file
+{{- end }}
+
+{{- define "copilot.config" -}}
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: flyte-data-config
+  namespace: {{`{{ namespace }}`}}
+data:
+  config.yaml: |
+    storage:
+{{- if eq .Values.storage.type "s3" }}
```

Comment:
Can't we do `{{ include storage }}` here?

Reply:
Agreed if that's possible we should avoid the duplication here.

Reply:
Hmm let me try

Reply:
Done!